### PR TITLE
feat: additional poll integrations (sqs, ebs, alb, elasticache)

### PR DESCRIPTION
**Pull request by woehrl01**
# Description

Please include a summary of the change and which issue is fixed (if relevant).

See #2253

Added:

* SQS
* Elasticache
* EBS
* ALB

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

Please delete options that are not relevant.

- [x] My commit message follows [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] My code is formatted to [Go standards](https://go.dev/blog/gofmt)
- [x] I have performed a self-review of my own code

## How to test this change?

Please describe how to test your changes. Include any relevant steps in the UI, HCL file(s), commands, etc

- Step 1
- Step 2
- etc

### CODE REVIEW

**Comment by woehrl01** [comment]
@NSSPKrishna anything missing to have this merged?

**Comment by Nandu-pns** [comment]
Hi @woehrl01, Thank you for your contribution on this, could you perhaps please include the tests for the newly added integrations ?

**Comment by woehrl01** [comment]
@Nandu-pns absolutely! I just added the tests, I also added missing ones for S3 and DocDb

**Comment by Nandu-pns** [comment]
Hi @woehrl01, Could you kindly correct the commit message since lints were failing ?

https://github.com/newrelic/terraform-provider-newrelic/actions/runs/4892645473/jobs/8734655914?pr=2289

**Comment by woehrl01** [comment]
@Nandu-pns done

**Comment by Nandu-pns** [comment]
Hi @woehrl01, Just a quick question, It appears like read context is missing these newly added integrations ? Could you please take a look whenever possible ? Thanks !

https://github.com/newrelic/terraform-provider-newrelic/blob/main/newrelic/resource_newrelic_cloud_aws_integrations.go#L613

**Review by Nandu-pns — APPROVED** [review]

File: newrelic/resource_newrelic_cloud_aws_integrations.go [hunk 1]
```diff
@@ -259,52 +384,111 @@ func expandCloudAwsIntegrationsInput(d *schema.ResourceData) (cloud.CloudIntegra
...
-	if x, ok := d.GetOk("x_ray"); ok {
-		cloudAwsIntegration.AwsXray = expandCloudAwsIntegrationXRayInput(x.([]interface{}), linkedAccountID)
-	} else if o, n := d.GetChange("x_ray"); len(n.([]interface{})) < len(o.([]interface{})) {
-		cloudDisableAwsIntegration.AwsXray = []cloud.CloudDisableAccountIntegrationInput{{LinkedAccountId: linkedAccountID}}
+	for key, fun := range awsIntegrationMap {
+		if v, ok := d.GetOk(key); ok {
+			fun.enableFunc(v.([]interface{}), linkedAccountID)
+		} else if o, n := d.GetChange(key); len(n.([]interface{})) > len(o.([]interface{})) {
+			fun.disableFunc(linkedAccountID)
+		}
```
Comment (pranav-new-relic) [inline newrelic/resource_newrelic_cloud_aws_integrations.go:491]: FYI (for the team's reference) - **please** ignore this comment

**Pull request merged**